docs/archive/project-json.md

```diff
@@ -37,6 +38,18 @@ The [`project.lock.json`](#projectlockjson) file (described below) is also used
 }
 ```
 
+## Migrate project.json to PackageReference
+
+The migration between project.json and PackageReference is straightforward. The easiest way to do it to use the built-in migrator in the latest Visual Studio 2022.
```

Comment:
We should probably have a note with the exact version of Visual Studio this is available in.